packages/fxa-settings/src/styles/tailwind.scss

```diff
@@ -5,10 +5,11 @@
 @import './animations.css';
 @import './ctas.scss';
 @import './unit-row.scss';
-@import './drop-down-menu.scss';
+
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+@tailwind screens;
```

Comment:
Is this needed for `@responsive` to work? If so maybe that's why it has to be moved in this file.

Reply:
It's needed both here and the separate file 👍 that's why we were having such weirdness with the responsive targetting 🤦